js/privatebin.js

```diff
@@ -1069,11 +1070,13 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
 
             // show text
             if (args !== null) {
-                // add jQuery object to it as first parameter
-                args.unshift($element.find(':last'));
-
-                // pass it to I18n
-                I18n._.apply(this, args);
+                // get last text node of element
+                var content = $element.contents();
+                if (content.length > 1) {
+                    content[content.length - 1].nodeValue = ' ' + I18n._(args);
```

Comment:
No man, don't introduce this space again. It just sucks… The space, I mean.

I changed `I18n._` in a way that it can lazy-load the missing translation of an element if the JSON file is not already loaded (which actually happened in my tests), that's why you **have to** to pass a jQuery element to it for full compatibility. (at least for items, which might be loaded at the start, and error/status messages certainly belong to them).